info.toml

```diff
@@ -216,10 +216,14 @@ path = "exercises/move_semantics/move_semantics5.rs"
 mode = "compile"
 hint = """
 Carefully reason about the range in which each mutable reference is in
-vogue. Does updating the value of referrent (x) immediately after the
-mutable reference is taken helps? Read more about 'Mutable Referenes'
+vogue. Does it help to update the value of referent (x) immediately after
```

Comment:
#1974 gets rid of `referent` altogether because it doesn't seem to help the explanation.